manifests/Better Vanilla Hai.yaml

```
@@ -0,0 +1,9 @@
+name: BetterVanillaHai
+authors: ZBok
+homepage: https://github.com/ZBok/BetterVanillaHai
+license: CC-BY-SA-4.0
+version: v0.9.1
+shortDescription: Adds Hai ships with a wide variety of human/Hai outfits to human fleets in the late-game. Spices up pirate combat and adds variety.
+description: Check the readme file for detailed information.
+url: https://github.com/ZBok/BetterVanillaHai/archive/refs/tags/v0.9.1.zip
+iconUrl: https://github.com/ZBok/BetterVanillaHai/blob/v0.9.1/icon.png
```

```suggestion
iconUrl: https://github.com/ZBok/BetterVanillaHai/raw/v0.9.1/icon.png
```
needs to be a direct link to an image, not github's view of an image